kythe/go/extractors/config/preprocessor/modifier/modifier_test.go

```diff
@@ -42,6 +43,7 @@ func TestBuildGradleHasKythe(t *testing.T) {
 		{fileName: "other-build.gradle", javacWrapper: "/tmp/javac-wrapper.sh", wantError: true},
 		// Look at other-build.gradle but use the correct javac wrapper.
 		{fileName: "other-build.gradle", javacWrapper: "/different/javac-wrapper.sh", hasWrapper: true},
+		{fileName: "other-build.gradle", javacWrapper: "/different/javac-wrapper.sh", hasWrapper: true},
```

Comment:
I forget what I was doing here, but this pull should be maven only, so axing this